content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 5
+description: Automatic instrumentation for opentelemetry-swift
+---
+
+OpenTelemetry-Swift provides several instrumentation libraries.
+
+## `SDKResourceExtension`
+
+`SDKResourceExtension` provides details about the device as a Resource.
+
+### Usage
+
+Use `DefaultResource.get()` to generate an all-in-one resource object. This
```

Comment:
Hmm, I'm a little confused. Does the installation of `SDKResourceExtension` cause `DefaultResource.get()` to generate all the data in the tables below for me? Or is there another API call I need to make?

Reply:
DefaultResource.get() will return a Resource object containing the data outlined below